AlfrescoKit/AlfrescoKit/View Controllers/Accounts View Controller/AKUserAccountListViewController.m

```diff
@@ -135,9 +136,12 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
         }
         
         __weak typeof(self) weakSelf = self;
-        [self.loginService loginToAccount:selectedAccount networkIdentifier:nil completionBlock:^(BOOL successful, id<AlfrescoSession> session, NSError *error) {
+        __block AlfrescoRequest *request = nil;
+        request = [self.loginService loginToAccount:selectedAccount networkIdentifier:nil completionBlock:^(BOOL successful, id<AlfrescoSession> session, NSError *error) {
+            [weakSelf.delegate controller:weakSelf didCompleteRequest:request error:error];
             [weakSelf.delegate userAccountListViewController:weakSelf didLoginSuccessfully:successful toAccount:selectedAccount creatingSession:session error:error];
         }];
+        [self.delegate controller:self didStartRequest:request];
```

Comment:
Ah yes, good point about the request object. Yes, I think a conditional check would be a good idea.